docs/src/examples/mpg_fr.adoc

```diff
@@ -13,66 +13,147 @@ trois incréments de pas: 0.1, 0.01, 0.001.
 
 Dans le fichier _custom.hal_ ou dans un fichier jog.hal, ajouter
 ce qui suit en vérifiant bien que les composants _mux4_ ou _encoder_ ne
-soient pas déjà utilisés. Si c'était le cas il faudrait en augmenter le nombre 
+soient pas déjà utilisés. Si c'était le cas il faudrait en augmenter le nombre
 en incrémentant la valeur du _count_ de la commande _loadrt_. Ajuster également
 le numéro de référence.
 Les composants mux4 et encoder sont décrits dans le manuel de HAL et dans les
 man pages.
 
+See the <<sec:hal-section,HAL Ini Section>> of the manual for more
+information on adding a hal file.
+
+Jog management pins are provided for each joint and all coordinate
+letters.  This example uses the axis jog pins for jogging in world
+mode.  Machines with non-identity kinematics may need use additional
+connections for jogging in joint mode.
+
+Manivelle de jog
 
-.Manivelle de jog
 ----
+# Jog Pendant
 loadrt encoder num_chan=1
 loadrt mux4 count=1
 addf encoder.capture-position servo-thread
 addf encoder.update-counters base-thread
 addf mux4.0 servo-thread
 
-# Mode position 
-# Chaque cran de manivelle provoque un pas calibré,
-# la durée du mouvement total peut dépasser la durée de rotation de la manivelle.
-# C'est le mode par défaut.
```

Comment:
Looks like very useful infos ! Shouldn't we add it to source instead ?

Reply:
Please do.

Reply:
done